CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 6.0.0 - 2022-01-18
+
+### Changed
+
+Upgrade axios version from ^0.21.1 to ^0.25.0. The difference in axios versions introduces several breaking changes as documented in their [change log](https://github.com/axios/axios/blob/master/CHANGELOG.md).
```

Comment:
I really don't like us saying "We've released a breaking change but we're not sure what that change is".

I don't know if breaking changes in axios to impact our users but this line will undoubtably cause concern and result in more questions in issues/changelogs for us to answer. We should identify if these changes will escape from our client library, and if so, we should decide whether that's something we want to have a breaking version for.

I had a skim through the axios PRs listed as breaking and I wouldn't expect them to affect us (eg - it's a breaking change to change the error when the user doesn't provide a URL, but that won't ever impact us). But it's hard to be sure. A lot of them are around typescript types, some were reverted in later versions, etc. But since we control the inputs and outputs to axios I would expect most of them to not reach our users.

I think this should probably be 5.1.1 and the description should mention that we've bumped the version to specifically upgrade an insecure subdependency.